event/src/unvalidated/event.rs

```diff
@@ -239,6 +239,18 @@ where
             )),
         }
     }
+
+    /// Get the CID of the init event of the stream for this event
+    pub fn stream_cid(&self) -> &Cid {
```

Comment:
maybe we should delete this? it's (now) a duplicate of the `id()` function that was recently modified